src/pages/RequestCallPage.js

```diff
@@ -82,10 +83,10 @@ class RequestCallPage extends Component {
         const {firstName, lastName} = this.getFirstAndLastName(props.myPersonalDetails);
         this.state = {
             firstName,
-            firstNameError: '',
+            firstNameError: false,
             lastName,
             phoneNumber: this.getPhoneNumber(props.user.loginList) || '',
-            lastNameError: '',
+            lastNameError: false,
```

Comment:
Ok. So lets rename these to `hasfirstNameError` and `haslastNameError` as these are booleans now. Also, do the same for other similar changes related to this PR.